geometry_msgs/Pose2D global_pose
FrenetMsg frenet_pose
int64 stop_line
bool lane_start
